integration/integration_test.go

```diff
@@ -919,6 +920,22 @@ func createGitServer(t *testing.T, opts gitServerOptions) *httptest.Server {
 	return httptest.NewServer(opts.authMW(gittest.NewServer(fs)))
 }
 
+func checkTestRegistry() {
+	resp, err := http.Get("http://localhost:5000/v2/_catalog")
+	if err != nil {
+		_, _ = fmt.Printf("Check test registry: %s\n", err.Error())
+		_, _ = fmt.Printf("Hint: Did you forget to run `make test-registry`?\n")
```

Comment:
```suggestion
		_, _ = fmt.Printf("Hint: Did you run `make test-registry`?\n")
```
Suggestion: This feels a bit more emphatic. Maybe I just didn't know 😏